dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -232,6 +237,34 @@ std::wstring Microsoft::Windows::AppNotifications::Helpers::SetDisplayNameBasedO
     return displayName;
 }
 
+// Placeholder
+HRESULT RetrieveAssetsFromProcess(_Out_ Microsoft::Windows::AppNotifications::Helpers::AppNotificationAssets& /*assets*/)
+{
+    // THROW_HR_IF_MSG(E_UNEXPECTED, VerifyIconFileExtension(iconFilePath));
+
+    return E_NOTIMPL;
+}
+
+// Do nothing. This is just a placeholder while the UDK is ingested with the proper API.
+HRESULT ToastNotifications_RetrieveAssets_Stub(_Out_ Microsoft::Windows::AppNotifications::Helpers::AppNotificationAssets& /*assets*/)
+{
+    // THROW_HR_IF_MSG(E_UNEXPECTED, VerifyIconFileExtension(iconFilePath));
+
+    return E_NOTIMPL;
+}
+
+HRESULT VerifyIconFileExtension(std::filesystem::path& iconFilePath)
+{
+    const auto fileExtension = iconFilePath.extension();
+
+    const bool isFileExtensionSupported =
```

Comment:
easier solution. Convert the extension to lower case. That takes care of .Bmp, .Png and other cases that aren't considered here. Then compare against the lower case variant. 

Or alternatively use wcscicmp or equivalent std:: API which ignores case